compose/compose-file/index.md

```diff
@@ -1506,42 +1506,54 @@ files](#volumes-for-services-swarms-and-stack-files).
 
 > **Note**: The top-level
 > [volumes](#volume-configuration-reference) key defines
-> a named volume and references it from each service's `volumes` list. This replaces `volumes_from` in earlier versions of the Compose file format. (See [Docker Volumes](/engine/userguide/dockervolumes.md) and
-[Volume Plugins](/engine/extend/plugins_volume.md) for general information on volumes.)
+> a named volume and references it from each service's `volumes` list. This replaces `volumes_from` in earlier versions of the Compose file format. See [Use Volumes](/engine/admin/volumes/volumes.md) and [Volume
+Plugins](/engine/extend/plugins_volume.md) for general information on volumes.
 
-This example shows a named volume (`db-data`) being used by the `postgres` service, and a mounted volume for a single service (under the `redis` service).
+This example shows a named volume (`mydata`) being used by the `web` service,
+and a mounted volume for a single service (under the `visualizer` service).
```

Comment:
The second one is a bind mount. It's always a bind mount if the first part (the source) is a path instead of just a name.